### PR TITLE
Feature/config flow

### PR DESCRIPTION
**Pull request by srippa-nexar**

### CODE REVIEW

File: src/config/__init__.py [hunk 1]
```diff
@@ -1,9 +1,2 @@
-from kitti_model_config import kitti_model_config
-from kitti_vgg16_config import kitti_vgg16_config
-from kitti_res50_config import kitti_res50_config
-from kitti_squeezeDet_config import kitti_squeezeDet_config
-from kitti_squeezeDetPlus_config import kitti_squeezeDetPlus_config
-from nexarear_squeezeDet_config import nexarear_squeezeDet_config
-from nexarear_squeezeDetPlus_config import nexarear_squeezeDetPlus_config
-from nexarear_res50_config import nexarear_res50_config
+from config import get_model_config
```
Comment (roiaddi) [inline src/config/__init__.py:1]: love it!!

File: src/nn_skeleton.py [hunk 2]
```diff
@@ -110,8 +100,73 @@ def __init__(self, mc):
     self.flop_counter = [] # array of tuple of layer name, flop number
     # activation counter
     self.activation_counter = [] # array of tuple of layer name, output activations
-    self.activation_counter.append(('input', mc.IMAGE_WIDTH*mc.IMAGE_HEIGHT*3))
+    self.activation_counter.append(('input', IMAGE_WIDTH*IMAGE_HEIGHT*3))
+
+  def _set_anchors_and_placeholders(self,preds):
```
Comment (roiaddi) [inline src/nn_skeleton.py:105]: not sure about this path after all

File: src/trainx.py [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+    det_class   = [det_class[idx] for idx in keep_idx]
+
+    _draw_box(
+        images[i], det_bbox,
+        [mc.dataset.CLASS_NAMES[idx]+': (%.2f)'% prob \
+            for idx, prob in zip(det_class, det_prob)],
+        (0, 0, 255))
+
+
+def train(mcfg):
```
Comment (roiaddi) [inline src/trainx.py:77]: what is the difference from train?

Comment (srippa-nexar) [reply src/trainx.py:77]: Just to minimize merge clashes

File: src/nn_skeleton.py (hunk 2 above)
Comment (srippa-nexar) [reply src/nn_skeleton.py:105]: If you refer to line 103, the It is not a path. It is a list counting number of elements in a layer.

**Pull request merged**